datafusion/expr/src/udaf.rs

```diff
@@ -276,6 +288,16 @@ pub trait AggregateUDFImpl: Debug + Send + Sync {
     fn create_groups_accumulator(&self) -> Result<Box<dyn GroupsAccumulator>> {
         not_impl_err!("GroupsAccumulator hasn't been implemented for {self:?} yet")
     }
+
+    /// Return the ordering expressions for the accumulator
+    fn sort_exprs(&self) -> Vec<Expr> {
```

Comment:
I don't understand how the sort_exprs could be supplied *by* the UDAF. I would expect them to be provided to the UDAF based on what was in the query